README.md

```diff
@@ -27,10 +27,26 @@ tracker](https://github.com/share/sharedb/issues).
 - Projections to select desired fields from documents and operations
 - Middleware for implementing access control and custom extensions
 - Ideal for use in browsers or on the server
-- Reconnection of document and query subscriptions
```

Comment:
Actually, as per #121, this is correct.

When the WebSocket reconnects, the [Connection middleware](https://github.com/share/sharedb/blob/master/lib/client/connection.js) reconnects both [documents](https://github.com/share/sharedb/blob/master/lib/client/connection.js#L302) and [query](https://github.com/share/sharedb/blob/master/lib/client/connection.js#L296) subscriptions.

We might need more clear documentation, but this line is correct.